examples/kms/src/main.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 The Tink-Rust Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+//! Example program demonstrating `tink-awskms`
+
+use tink::{keyset::insecure, registry::KmsClient, AeadBoxClone};
+
+const KEY_URI: &str =
+    "aws-kms://arn:aws:kms:us-east-2:235739564943:key/3ee50705-5a82-4f5b-9753-05c4f473922f";
+const CRED_INI_FILE: &str = "../../../testdata/credentials_aws.ini";
+
+fn main() {
+    tink_aead::init();
+
+    // Generate a new key.
+    let kh1 = tink::keyset::Handle::new(&tink_aead::aes256_gcm_key_template()).unwrap();
+
+    // Set up the main key-encryption key at a KMS. This is an AEAD which will generate a new
+    // data-encryption key (DEK) for each encryption operation; the DEK is included in the
+    // ciphertext emitted from the encryption operation, in encrypted form (encrypted by the
+    // KMS main key).
+    let kms_client = tink_awskms::AwsClient::new_with_credentials(KEY_URI, CRED_INI_FILE).unwrap();
+    let backend = kms_client.get_aead(KEY_URI).unwrap();
+    let main_key = Box::new(tink_aead::KmsEnvelopeAead::new(
+        tink_aead::aes256_gcm_key_template(),
+        backend,
+    ));
+
+    // An implementation of `keyset::Reader` and `keyset::Writer` which simply writes to memory.
+    let mut mem_keyset = tink::keyset::MemReaderWriter::default();
+
+    // Write encrypts the keyset handle with the main key and writes to the `keyset::Writer`
```

Comment:
These comments are a bit difficult to parse when reading it for the first time. Perhaps something like "Use `write` to encrypt ..."? Also perhaps something similar for the "Read" comment below.

Reply:
Yeah, I've clarified a bit but these comments (from [upstream](https://github.com/google/tink/blob/master/docs/GOLANG-HOWTO.md#storing-and-loading-existing-keysets)) could do with more &ndash; will do.